memgpt/main.py

```diff
@@ -255,14 +256,14 @@ async def main(
 
         human_persona = human
         if human_persona is None:
-            human_persona = (humans.DEFAULT, None)
+            human_persona = humans.DEFAULT
         else:
             try:
                 humans.get_human_text(human_persona, Config.custom_humans_dir)
                 human_persona = (human_persona, Config.custom_humans_dir)
             except FileNotFoundError:
                 humans.get_human_text(human_persona)
-                human_persona = (human_persona, None)
+                human_persona = human_persona

```

Comment:
Doesn't this need to be a tuple?

Reply:
The following code did not use a tuple, that is why the IDE was mourning. But I may have made the change in the wrong place.

Reply:
Skimming the code it seems that there can be a tuple of a string, while a tuple denotes a persona string + a directory path string. I *guess* that somewhere the value can be either and that a tuple with None as second parameter is basically identical with just the string. But I may be wrong about that.